send_status_email

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+from textwrap import dedent
+
+from ocflib.misc.mail import send_mail
+
+# TSV or space separated file of username \t # of unexcused absences
+# e.g. abizer   2
+DATA_FILE = sys.argv[1]
+
+SUBJECT = '[UNIX SysAdmin Decal] Attendance and Lab Report'
+FROM = 'decal@ocf.berkeley.edu'
+CC = 'decal@ocf.berkeley.edu'
+
+message = dedent('''
+Hello {name},
+
+This is a status update for your current standing in the course.
+
+Our records indicate you have {absences} unexcused absences. As a reminder, more than two unexcused absences is
+grounds for NPing the course. Note that this figure does not include the final survey, which will allow you to waive
+one of the unexcused absences. You can find a link to the survey on Piazza.
+
+Our records indicate that you have been checked off for the following labs:
+
+{labs}
+
+Please note that this does not include labs that you have submitted but are pending to be checked off by a facilitator.
+As a reminder, you need to complete at least 8/10 labs to pass the course.
```

Comment:
8/10? I thought we were making them do all the labs. 8/10 was last sem, IIRC.

Reply:
no, last semester was 6/10, and this semester was 8/10